py3status/modules/dropboxd_status.py

```diff
@@ -86,14 +86,16 @@ def dropbox(self):
 
         if status == "Dropbox isn't running!":
             color = self.py3.COLOR_BAD
-            status = self.status_off
+            _format = self.status_off
         elif status == "Up to date":
             color = self.py3.COLOR_GOOD
-            status = self.status_on
+            _format = self.status_on
         else:
             color = self.py3.COLOR_DEGRADED
             if self.status_busy is not None:
-                status = self.status_busy
+                _format = self.status_busy
+
+        status = self.py3.safe_format(_format, {'status': status})
```

Comment:
Full disclose: I have this too. https://github.com/lasers/py3status/blob/newformat-dropbox/py3status/modules/dropboxd_status.py and some more modules like it.

It's awesome and fun project for me. We don't have to merge them to keep things as-is (states, statuses, etc). Notice the new lines if we stopped using `[0]`.